codereadyworkspaces/stack/Dockerfile

```diff
@@ -0,0 +1,67 @@
+FROM registry.redhat.io/codeready-workspaces/plugin-java8-rhel8:2.10-4
+
+ENV ARGOCD_VERSION=2.0.5
+ENV YQ_VERSION=v4.11.1
+ENV HELM_VERSION=v3.6.0
+ENV KUBESEAL_VERSION=v0.13.1
+ENV TEKTON_VERSION=0.20.0
+ENV OC_VERSION=4.8.3
+ENV JQ_VERSION=1.6
+ENV CONFTEST_VERSION=0.23.0
+ENV ZSH_VERSION=5.8
+
+USER root
+
+# argo
+RUN curl -sL https://github.com/argoproj/argo-cd/releases/download/v${ARGOCD_VERSION}/argocd-linux-amd64 -o /usr/local/bin/argocd && \
+    chmod -R 775 /usr/local/bin/argocd && \
+    echo "🐙🐙🐙🐙🐙"
+
+#helm
+RUN  curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/master/scripts/get-helm-3 && \
+     chmod 700 get_helm.sh && \
+     ./get_helm.sh --version ${HELM_VERSION} && \
+     echo "⚓️⚓️⚓️⚓️⚓️"
+
+#🦭🦭🦭kubeseal 🦭
+RUN curl -sL https://github.com/bitnami-labs/sealed-secrets/releases/download/${KUBESEAL_VERSION}/kubeseal-linux-amd64 -o /usr/local/bin/kubeseal && \
+    chmod -R 755 /usr/local/bin/kubeseal && \
+    echo "🦭🦭🦭🦭🦭"
+
+# tekton
+RUN curl -sLO https://github.com/tektoncd/cli/releases/download/v${TEKTON_VERSION}/tkn_${TEKTON_VERSION}_Linux_x86_64.tar.gz && \
+    tar xvzf tkn_${TEKTON_VERSION}_Linux_x86_64.tar.gz -C /usr/local/bin/ tkn  && \
```

Comment:
this failed for me ? probs the space 🦤 before tkn

may be useful to use the OpenShift matching version from the mirror:
```
ENV TEKTON_VERSION=0.19.1

# tekton
RUN curl -sL https://mirror.openshift.com/pub/openshift-v4/clients/pipeline/latest/tkn-linux-amd64-${TEKTON_VERSION}.tar.gz | tar --strip-components=1 -C /usr/local/bin -xzf - tkn && \
    echo "🐈🐈🐈🐈🐈"
```